docs/references/sql-introspection.md

```diff
@@ -0,0 +1,129 @@
+| `trace_id` | `Utf8` | The ID of the trace that is assigned to this invocation. Only relevant when tracing is enabled. |
+| `journal_size` | `UInt32` | The number of journal entries durably logged for this invocation. |
+| `created_at` | `Date64` | Timestamp indicating the start of this invocation. |
+| `modified_at` | `Date64` | Timestamp indicating the last state transition. For example, last time the status changed from `invoked` to `suspended`. |
+| `retry_count` | `UInt64` | The number of attempts since the last successful attempt of this invocation. Increments on start, so 2 or more means a failure occurred. |
+| `last_start_at` | `Date64` | Timestamp indicating the start of the most recent attempt of this invocation. |
+| `next_retry_at` | `Date64` | Timestamp indicating the start of the next attempt of this invocation. |
+| `last_attempt_deployment_id` | `Utf8` | The opaque service deployment ID that was used in the most recent attempt of this invocation; this will be set before a journal entry is stored, but can change later. |
+| `last_attempt_server` | `Utf8` | Server/SDK version, e.g. `restate-sdk-java/1.0.1` |
+| `last_failure` | `Utf8` | An error message describing the most recent failed attempt of this invocation, if any. |
+| `last_failure_error_code` | `Utf8` | The error code of the most recent failed attempt of this invocation, if any. |
+| `last_failure_related_entry_index` | `UInt64` | The index of the journal entry that caused the failure, if any. It may be out-of-bound of the currently stored entries in `sys_journal`. |
+| `last_failure_related_entry_name` | `Utf8` | The name of the journal entry that caused the failure, if any. |
+| `last_failure_related_entry_type` | `Utf8` | The type of the journal entry that caused the failure, if any. You can check all the available entry types in [`entries.rs`](https://github.com/restatedev/restate/blob/main/crates/types/src/journal/entries.rs). |
+| `status` | `Utf8` | Either `pending` or `ready` or `running` or `backing-off` or `suspended` or `completed`. |
+
+## Table: `sys_journal`
+
+| Column name | Type | Description |
+|-------------|------|-------------|
+| `partition_key` | `UInt64` | Internal column that is used for partitioning the services invocations. Can be ignored. |
+| `id` | `Utf8` | [Invocation ID](/operate/invocation#invocation-identifier). |
+| `index` | `UInt32` | The index of this journal entry. |
+| `entry_type` | `Utf8` | The entry type. You can check all the available entry types in [`entries.rs`](https://github.com/restatedev/restate/blob/main/crates/types/src/journal/entries.rs). |
+| `name` | `Utf8` | The name of the entry supplied by the user, if any. |
+| `completed` | `Boolean` | Indicates whether this journal entry has been completed; this is only valid for some entry types. |
+| `invoked_id` | `Utf8` | If this entry represents an outbound invocation, indicates the ID of that invocation. |
```

Comment:
Shouldn't this be inbound invocation? 
`The ID of the inbound invocation: the original invocation that came in over the Restate ingress (HTTP/Kafka/SDK clients).`
Maybe needs some more iterations

Reply:
What this column means is if the journal entry is a call/send, then the `invoked_id` will contain its invocation id. So outbound invocation is meant to be call and send (one way call).